<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimHitControllerForm</class>
 <widget class="QWidget" name="SimHitControllerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sim Hits</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lblGeneral">
           <property name="text">
            <string>Select hits to show</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>71</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Inner Detector</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="chbxPixelHits">
              <property name="text">
               <string>Pixel Hits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chbxSCTHits">
              <property name="text">
               <string>SCT Hits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chbxTRTHits">
              <property name="text">
               <string>TRT Hits</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Calorimeters</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QCheckBox" name="chbxLArEMBHits">
              <property name="text">
               <string>LAr EMB Hits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chbxLArEMECHits">
              <property name="text">
               <string>LAr EMEC Hits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chbxLArFCALHits">
              <property name="text">
               <string>LAr FCAL Hits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chbxLArHECHits">
              <property name="text">
               <string>LAr HEC Hits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chbxTileHits">
              <property name="text">
               <string>Tile Hits</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>MuonSpectrometer</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="chbxMDTHits">
              <property name="text">
               <string>MDT Hits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chbxRPCHits">
              <property name="text">
               <string>RPC Hits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chbxTGCHits">
              <property name="text">
               <string>TGC Hits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chbxCSCHits">
              <property name="text">
               <string>CSC Hits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chbxMMHits">
              <property name="text">
               <string>MM Hits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chbxsTGCHits">
              <property name="text">
               <string>sTGC Hits</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chbxGenericMuonHits">
              <property name="text">
               <string>Generic</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Forward Region</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="chbxForwardRegionHits">
          <property name="text">
           <string>Forward Region</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>111</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>111</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
